jaxrs/src/test/java/com/cerner/beadledom/jaxrs/provider/FakeModel.java

```diff
@@ -28,6 +28,10 @@
   @JsonProperty("inner_models")
   public List<FakeInnerModel> innerModels;
 
+  public FakeModel() {
```

Comment:
The default constructer and the setters were added to make the creating of the FakeModel more like the builder pattern. It is used in the Spec at the recommendation of Brian.